tests/test_generators.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
```

Comment:
As you are not mocking anything here you can drop the imports. Once you do this the CI will pass.